src/components/atoms/OcDrop/OcDrop.vue

```diff
@@ -207,6 +207,28 @@ export default {
   .tippy-content {
     // note: needed so that the box shadow from `oc-box-shadow-medium` doesn't get suppressed
     padding: 8px;
+
+    li {
+      padding-left: 0 !important;
```

Comment:
I don't agree with the two paddings on **any** `li` in an `OcDrop`. That component is quite generic and it doesn't have the sole purpose of serving a context menu.

What do you think about adding a `ContextDrop` or `ContextMenu` component to `web` instead, which then internally uses an `OcDrop` in a standardized way?